Assets/Tests2/NewTestScript.cs

```diff
@@ -0,0 +1,320 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+using AppsFlyerSDK;
+using NSubstitute;
+
+namespace AppsFlyerSDK.Tests
+{
+    public class NewTestScript
+    {
+
+
+        [Test]
+        public void testStartSDK()
+        {
+            var AppsFlyerMOCKInterface = Substitute.For<IAppsFlyerNativeBridge>();
+            AppsFlyer.instance = AppsFlyerMOCKInterface;
+            AppsFlyer.startSDK();
+            AppsFlyerMOCKInterface.Received().startSDK(Arg.Any<bool>(), Arg.Any<string>());
+           
+        }
+
+        [Test]
+        public void testSendEvent()
```

Comment:
Because we test in first 2 tests common behavior, worth to change the test name accordingly

`testSendEvent` -> `testSendEventIsCalled`